<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallDwfsDialog</class>
 <widget class="QDialog" name="CallDwfsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dwfs frontend</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Solver parameter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Case name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="leCase"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Solution type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="cbSimulation"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Directory</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leWorkDir"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pbBrowse">
        <property name="text">
         <string>&amp;Browse...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QRadioButton" name="rbVelocity">
          <property name="text">
           <string>Velocity</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="sbVelocity">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>999999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>10.000000000000000</double>
          </property>
          <property name="value">
           <double>50.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Alpha</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QDoubleSpinBox" name="sbAlpha">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <double>-89.989999999999995</double>
          </property>
          <property name="maximum">
           <double>89.989999999999995</double>
          </property>
          <property name="singleStep">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="rbMach">
          <property name="text">
           <string>Mach number</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="sbMach">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <double>0.010000000000000</double>
          </property>
          <property name="maximum">
           <double>0.990000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Beta</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QDoubleSpinBox" name="sbBeta">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <double>-89.989999999999995</double>
          </property>
          <property name="maximum">
           <double>89.989999999999995</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pbChangeSolver">
          <property name="text">
           <string>Change solver</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>158</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbLoadConfig">
          <property name="text">
           <string>&amp;Load...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbEditConfig">
          <property name="text">
           <string>&amp;Edit...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Reference dimensions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string> Area</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QScienceSpinBox" name="sbRefArea">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Reference point X</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QScienceSpinBox" name="sbRefX">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QScienceSpinBox" name="sbRefSpan">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Reference point Y</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QScienceSpinBox" name="sbRefY">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Chord</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QScienceSpinBox" name="sbRefChord">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Reference point Z</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QScienceSpinBox" name="sbRefZ">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>-999999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Span</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>License </string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Host ID:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>License Key:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="leLicenseKey">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lbHostID">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbStart">
       <property name="text">
        <string>&amp;Start solver</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QScienceSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../qsciencespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leCase</tabstop>
  <tabstop>cbSimulation</tabstop>
  <tabstop>leWorkDir</tabstop>
  <tabstop>pbBrowse</tabstop>
  <tabstop>rbVelocity</tabstop>
  <tabstop>sbVelocity</tabstop>
  <tabstop>sbAlpha</tabstop>
  <tabstop>rbMach</tabstop>
  <tabstop>sbMach</tabstop>
  <tabstop>sbBeta</tabstop>
  <tabstop>pbLoadConfig</tabstop>
  <tabstop>pbEditConfig</tabstop>
  <tabstop>sbRefArea</tabstop>
  <tabstop>sbRefSpan</tabstop>
  <tabstop>sbRefChord</tabstop>
  <tabstop>sbRefX</tabstop>
  <tabstop>sbRefY</tabstop>
  <tabstop>sbRefZ</tabstop>
  <tabstop>leLicenseKey</tabstop>
  <tabstop>pbStart</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>rbVelocity</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbVelocity</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbMach</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbMach</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>CallDwfsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
